Add Figures, Tables and Equations Guidelines (#13)
SOMAS2021/SOMAS-Final-Report: README.md

You should review your writing for correct spelling, grammar, and punctuation.

## Figures, Tables and Equations

> Figures should add to the text. Redundant figures should be removed since they can confuse the reader and distract them from the important points in the text.

Reviewer: ```suggestion
Figures should add to the text. Do not add redundant figures since they can confuse the reader and distract them from the important points in the text.
```

> Figures, tables and similar objects should always be talked about in the text. They also need to be refered to by figure number and not position.

Reviewer: ```suggestion
Figures, tables and similar objects should always be talked about in the text. They also need to be referred to by figure number and not position. Add a label to your figure using `\label{fig:YOUR_LABEL_HERE}` and refer to it in text using `Figure \ref{fig:YOUR_LABEL_HERE}`.
```

e.g. 
- Good - "as you can see in Figure 6"
- Bad  - "as you can see in the figure below"

> Equations should be refered to using their equation number with parenthesis around the equation number.

Reviewer: ```suggestion
Equations should be refered to using their equation number with parenthesis around the equation number. Remember to add labels and use references when writing so you don't have to manually renumber things.
```
